### PR TITLE
Time embedding

### PR DESCRIPTION
**Pull request by gkondas**
## Summary by CodeRabbit

* **New Features**
  * Added support for optional per-sample duration values that flow through the data pipeline and are incorporated into model embeddings.

* **Bug Fixes**
  * Improved null-value handling in censored column filtering logic.

### CODE REVIEW

File: src/every_query/model.py [hunk 1]
```diff
@@ -421,9 +425,21 @@ def _hf_inputs(self, batch: EveryQueryBatch) -> dict[str, torch.Tensor]:
         Returns:
             A dictionary of inputs for the Hugging Face model.
         """
+        attention_mask = batch.code != batch.PAD_INDEX  # (batch_size, seq_len)
+
+        if batch.duration_days is not None:
+            word_embeds = self.HF_model.embeddings.word_embeddings(batch.code)  # (B, seq_len, H)
+            dur_norm = (batch.duration_days / 365.0).unsqueeze(-1)  # (B, 1)
```
Comment (payalchandak) [inline src/every_query/model.py:432]: why normalize before embedding?
